backend/backend/api/routes.py

```diff
@@ -9,6 +9,9 @@
 from utils.logger import logger
 from .schemas import QueryRequestSchema, EventsRequestSchema, ResponseSchema
 
+import gradio as gr
```

Comment:
Consider importing the necessary dependencies at the top of the file for better organization.